ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -58,4 +58,5 @@ bash -c ' \
     REPO_URL="http://download.ceph.com/rpm-${CEPH_VERSION}/el__ENV_[BASEOS_TAG]__/"; \
   fi && \
   rpm -Uvh "$REPO_URL/noarch/ceph-release-1-${RELEASE_VER}.el__ENV_[BASEOS_TAG]__.noarch.rpm" ' && \
-yum install -y __CEPH_BASE_PACKAGES__
+yum install -y __CEPH_BASE_PACKAGES__ && \
+python3 -m pip install -U scikit-learn
```

Comment:
```suggestion
python3 -m pip install -U scikit-learn==0.19.2
```

To keep the same requirement than the one from the ceph sources.

https://github.com/ceph/ceph/blob/master/src/pybind/mgr/diskprediction_local/requirements.txt#L3

Reply:
done!